articles/how-to-send-sms-in-django-using-twilio/index.md

```diff
@@ -0,0 +1,230 @@
+SMS is a good way to send alerts, notifications and even authenticate users. There comes a time where by we need sms functionality in our app, either to send notifications, send sms to clients or even serve as a way of having a "multifactor authentication" feature.
+
+In this tutorial, we will build a simple Django project with SMS capability that sends message to a phone number once a particular condition is met.
+
+Twillio is a cloud communications platform that allows you to add voice and SMS functionality in your app . Its actually a paid platform, but for the sake of this tutorial we'll be making use of the trial account.
+
+### Prerequisites
+To follow along with this tutorial, you'll need [Python3](https://www.python.org/downloads/) installed on your machine.
+
+A basic understanding of Django would help the reader follow along better.
+
+### Table of contents
+- [Creating and setting up a new Django project](#creating-and-setting-up-a-new-django-project)
+- [Setting up Twilio](#setting-up-twilio)
+- [Creating a model](#creating-a-model)
+- [Configuration](#configuration)
+- [Conclusion](#conclusion)
+
+
+
+### Creating and setting up a new Django project
+Let's start by creating a new virtual environment.
+
+A virtual environment allows you to create different spaces on your computer, with a different set of libraries and versions.
+
+By creating a virtual environment, you'll be able to separate the necessary library installation for a project, without having to install them globally.
+
+Now, you create a virtual environment `env` as shown below:
+
+```bash
+$ python -m venv /path/to/new/virtual/environment
+```
+
+Here, we specify the `/path/to/new/virtual/environment` as `env`.
+
+On creation, you can activate the virtual environment using the following command:
+
+```bash
+$ source env/bin/activate
+```
+
+On activating the environment, we can install Django using the following command:
```

Comment:
Don't mix pronouns. Choose one and stick to it throughout your tutorial. Ie. Choose between `you` and `we` and use it throughout.